trackpy/feature.py

```diff
@@ -207,6 +208,7 @@ def _refine(raw_image, image, radius, coords, max_iterations,
     GOOD_ENOUGH_THRESH = 0.005
 
     ndim = image.ndim
+    anisotropic = radius[1:] != radius[:-1]
```

Comment:
I am giving myself a hard time in doing this pr... thanks for all your
feedback :+1: